soh/src/overlays/actors/ovl_En_Mag/z_en_mag.c

```diff
@@ -737,13 +725,12 @@ void EnMag_DrawInnerMq(Actor* thisx, PlayState* play, Gfx** gfxp) {
     gDPSetPrimColor(gfx++, 0, 0, (s16)this->copyrightAlpha, (s16)this->copyrightAlpha, (s16)this->copyrightAlpha,
                     (s16)this->copyrightAlpha);
 
-    if ((s16)this->copyrightAlpha != 0) 
-    {
-        gDPLoadTextureBlock(gfx++, EnMag_GetCopyrightTex(), G_IM_FMT_IA, G_IM_SIZ_8b, 128, 16, 0,
+    if ((s16)this->copyrightAlpha != 0) {
+        gDPLoadTextureBlock(gfx++, copy_tex, G_IM_FMT_IA, G_IM_SIZ_8b, copy_width, 16, 0,
                             G_TX_NOMIRROR | G_TX_CLAMP, G_TX_NOMIRROR | G_TX_CLAMP, G_TX_NOMASK, G_TX_NOMASK,
                             G_TX_NOLOD, G_TX_NOLOD);
 
-        gSPTextureRectangle(gfx++, 312, 792, 952, 856, G_TX_RENDERTILE, 0, 0, 1 << 10, 1 << 10);
+        gSPTextureRectangle(gfx++, copy_xl, 792, copy_xh, 856, G_TX_RENDERTILE, 0, 0, 1 << 10, 1 << 10);
```

Comment:
feels like this PR is doing way more than it should, it looks like we already had some logic for 1998 vs 1998/2003, and we just need to set the different locations/lengths here

Reply:
It does exactly what is needed, only four values change, and since they only change per version, there is no need to run a function for each value.